sdk/Lusid.Sdk.Tests/tutorials/Ibor/StructuredResultStore.cs

```diff
@@ -0,0 +1,946 @@
+                principal: 100m,
+                couponRate: 0.06m,
+                flowConventions: new FlowConventions(
+                    currency: "GBP",
+                    paymentFrequency: "6M",
+                    rollConvention: "MF",
+                    dayCountConvention: "Act365",
+                    paymentCalendars: new List<string>(),
+                    resetCalendars: new List<string>(),
+                    settleDays: 2,
+                    resetDays: 2),
+                identifiers: new Dictionary<string, string>(),
+                instrumentType: LusidInstrument.InstrumentTypeEnum.Bond
+            );
+            
+            // Upsert the created bond.
+            var instrumentsIds = new List<(LusidInstrument, string)>
+            {
+                (bond, bond.InstrumentType + Guid.NewGuid().ToString())
+            };
+            var definitions = TestDataUtilities.BuildInstrumentUpsertRequest(instrumentsIds);
+            var upsertResponse = _instrumentsApi.UpsertInstruments(definitions);
+            
+            // Create transaction and upsert transcation onto the portfolio.
+            List<string> luids = upsertResponse.Values
+                .Select(inst => inst.Value.LusidInstrumentId)
+                .ToList();
+            var transactionRequest = TestDataUtilities.BuildTransactionRequest(luids, effectiveAt);
+            _transactionPortfoliosApi.UpsertTransactions(scope, portfolioCode, transactionRequest);
+            
+            // Create and upsert Data Map, indicating what data will be passed in via the document.
+            // CompositeLeaf is created, it does not represent a column in the document, it links Accrual/Amount and Accrual/Ccy. 
+            // When creating CompositeLeaf no name should be used (should be null).
+            DataMapping dataMapping = new DataMapping(new List<DataDefinition>
+            {
+                new DataDefinition("UnitResult/LusidInstrumentId", "LusidInstrumentId", "string", "Unique"),
+                new DataDefinition("UnitResult/Accrual", dataType: "Result0D", keyType: "CompositeLeaf"),
+                new DataDefinition("UnitResult/Accrual/Amount", "Accrual", "decimal", "Leaf"),
+                new DataDefinition("UnitResult/Accrual/Ccy", "AccrualCcy", "string", "Leaf"),
+                new DataDefinition("UnitResult/ClientCustomValue", "ClientVal", "decimal", "Leaf"),
+            });
+            var request = new CreateDataMapRequest(dataMapKey, dataMapping);
+            _structuredResultDataApi.CreateDataMap(scope,
+                new Dictionary<string, CreateDataMapRequest> {{"dataMapKey", request}});
+            
+            // Upsert Document containing client data. This data contains normalised accrual but not PV.
+            string document = $"LusidInstrumentId, Accrual, AccrualCcy, ClientVal\n{luids.First()}, 0.0123456, GBP, 1.7320508"; // Note the LusidInstrumentId the previously defined instrument.
```

Comment:
I think it would be worth explaining you've created a csv document. Is the format significant or can it be able format? Need to explain what is valid